lib/ansible/plugins/callback/unixy.py

```diff
@@ -128,9 +136,15 @@ def v2_runner_on_ok(self, result, msg="ok", display_color=C.COLOR_OK):
         task_result = self._process_result_output(result, msg)
         self._display.display("  " + task_result, display_color)
 
+    def v2_runner_item_on_skipped(self, result):
+        self.v2_runner_on_skipped(result)
+
     def v2_runner_item_on_failed(self, result):
         self.v2_runner_on_failed(result)
 
+    def v2_runner_item_on_ok(self, result):
+        self.v2_runner_on_ok(result)
```

Comment:
not a blocker, but 'no color in items'?

Reply:
I'd be glad to fix this but I'm not sure what you mean. I do get colorful output for the items: ![screenshot](https://imgur.com/a/6zbW0tB)

Reply:
nvmd, had not looked at the called method which already handles skipped coloring